### PR TITLE
Developer setup documentation update

### PR DESCRIPTION
**Pull request by jachro**
addresses #501

### CODE REVIEW

File: docs/developer/python_setup.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    $ python -version
+
+If you have python 2 you can consider uninstalling it with
+
+.. code-block:: console
+
+    $ brew uninstall --force python@2
+
+Before installing Python 3 add a following export line to ``~/.profile``
```
Comment (rokroskar) [inline docs/developer/python_setup.rst]:
```suggestion
Before installing Python 3 add the following line to ``~/.profile``
```

File: docs/developer/python_setup.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+.. code-block:: console
+
+    $ export PATH="/usr/local/opt/python/libexec/bin:$PATH"
```
Comment (rokroskar) [inline docs/developer/python_setup.rst]: I don't have this in my `PATH` -- why is it needed?

File: docs/developer/setup.rst [hunk 3]
```diff
@@ -79,6 +84,25 @@ Once minikube is started, make sure you can access it by running:
...
+    $ minikube addons enable coredns
+    $ eval $(minikube docker-env)
+
+Installing Chartpress
+^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: console
+
+    $ pip install chartpress
+    $ chartpress --tag latest
```
Comment (rokroskar) [inline docs/developer/setup.rst]: This isn't about installing chartpress, but about building the images. So this is probably unnecessary here. But you may say two words at the beginning of this section about chartpress, e.g.

"We use the `chartpress` package to bundle together images and charts."

File: docs/developer/python_setup.rst (hunk 2 above)
Comment (jachro) [reply docs/developer/python_setup.rst]: I'm not entirely sure but I think this is to override the default osx' python so the brew installed python is kind of hiding the system one.

File: docs/developer/setup.rst (hunk 3 above)
Comment (jachro) [reply docs/developer/setup.rst]: Yes, that's true but I guess `pip install chartpress` should stay here maybe?

File: docs/developer/python_setup.rst (hunk 2 above)
Comment (jirikuncar) [reply docs/developer/python_setup.rst]: IMHO it's not necessary. If there is a problem with Python binary installed by brew, one should just run `brew doctor`.

File: docs/developer/setup.rst (hunk 3 above)
Comment (rokroskar) [reply docs/developer/setup.rst]: Actually, we use `pipenv` anyway with the provided `Pipfile` -- that's what the `make minikube-deploy` will use. So this section is not really necessary IMO.

Comment (jachro) [reply docs/developer/setup.rst]: I'm sure you know better how the Python machinery works here. I took the section out.

File: docs/developer/python_setup.rst (hunk 2 above)
Comment (rokroskar) [reply docs/developer/python_setup.rst]: I think @jirikuncar's point is that brew should take care of making sure that the brew-installed versions are the ones that are actually used. But you might be right that it doesn't always work out that way...

Comment (jachro) [reply docs/developer/python_setup.rst]: My experience is that when it comes into binaries both provided by OSX natively and installed with `brew`, the default is that the system ones are winning as they are on the `PATH` and the additional ones not necessarily. And the problem is that for `brew` everything is fine as from its perfectly and installation wise all is good. But maybe I'm doing something wrong.

**Comment by rokroskar** [comment]
Thanks @jachro! Please squash before merging.

**Pull request merged**